lib/openglvis.cpp

```diff
@@ -66,10 +66,10 @@ void Camera::GLMultRotMatrix()
    double mat[16] =
    {
       -left[0], up[0], -dir[0], 0.0,
-         -left[1], up[1], -dir[1], 0.0,
```

Comment:
Interesting, when I run `make style` it puts these indents in... what version of *astyle* are you using?

Reply:
Version 2.05.1, see https://github.com/mfem/mfem/blob/master/CONTRIBUTING.md#developer-guidelines :)